<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkOptionWindow</class>
 <widget class="QDialog" name="NetworkOptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>155</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>285</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="connectButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLabel" name="portNumberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Port Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hostAddressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Host Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="portNumberCombo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5234</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4000</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="locationCombo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>133</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>北京路</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>hostAddressLineEdit</tabstop>
  <tabstop>portNumberCombo</tabstop>
  <tabstop>locationCombo</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
